lib/src/tasks/coverage/exceptions.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
grr.. I originally added `coverage` to the gitignore to ignore the generated report, which erroneously started ignoring new files added to this `coverage` directory. I thought I had fixed this but I'll check again